docs/Release-v2.1.0-Database-Migration.md

```diff
@@ -59,10 +59,11 @@ command. This option may be useful for instance to test out the
 migration to get a sense of the running time and increased MongoDB
 storage requirements for your typical datasets.
 
-1. Increase the storage capacity of your MongoDB to account for the
+1. Make sure that all users/runners have upgraded their SDKs!
+2. Increase the storage capacity of your MongoDB to account for the
 increased disk usage of your Datasets. Monitor this usage over time as
 datasets are migrated.
-2. Follow Option 1 or Option 3 below:
+3. Follow Option 1 or Option 3 below:
```

Comment:
@allenleetc as discussed offline, Option 1 is not actually valid right? Only SDKs with `FIFTYONE_DATABASE_ADMIN=true` will cause the datasets to be lazily migrated. Individual users won't have this, and *I believe* that the central Teams deployment also no longer runs as admin? <- needs aloha validation